projects/core/src/product/services/product-loading.service.spec.ts

```diff
@@ -328,9 +335,12 @@ describe('ProductLoadingService', () => {
         Expect:
           - Trigger emission at 80ms (closest subscription to maxAge expiration)
          */
-        const subscriber$ = timer(0, 40, getTestScheduler()).pipe(
-          take(3),
-          switchMap((intervalId) => (intervalId % 2 ? NEVER : trigger$))
+        const VALUE = 2;
```

Comment:
In case with multiple, independent initializations, you can use 
`const VALUE = 2, COUNT = 3, PERIOD = 40;`

Reply:
this gives error in ts lint

Reply:
done